OA-PASS/pass-docker
# Bump indexer to 0.0.8, bump Fedora tag

# Overview
* Uses the latest indexer, which is compatible with model 2.1
* Bump Fedora tag to contain `-2.1`.

# To test
Point the 2.1-compatible indexer integration tests to 
`oapass/fcrepo:4.7.5-2.1-SNAPSHOT`
`oapass/indexer:0.0.8-2.1-SNAPSHOT`
... and see if ITs start passing

Interested parties:
@karenhanson
